mimic/test/test_maas.py

```diff
@@ -98,11 +113,10 @@ def createAlarm(self, label, entity_id, check_id):
         postdata['notification_plan_id'] = 'npTechnicalContactsEmail'
         alarms_endpoint = '{0}/entities/{1}/alarms'.format(self.uri, entity_id)
         req = request(self, self.root, b"POST", alarms_endpoint,
-                      json.dumps(postdata))
+                      json.dumps(postdata).encode("utf-8"))
         resp = self.successResultOf(req)
         self.assertEquals(resp.code, 201)
-        alarm_id = resp.headers.getRawHeaders('x-object-id')[0]
-        location = resp.headers.getRawHeaders('location')[0]
+        alarm_id, location = id_and_location(resp)
```

Comment:
no problem, it was just too tedious to retype again at the numerous call sites :)